Tests/PostgresNIOTests/New/IntegrationTests.swift

```diff
@@ -0,0 +1,328 @@
+//
+//  File.swift
+//  
+//
+//  Created by Fabian Fett on 13.01.21.
+//
+
+import Logging
```

Comment:
Ditch Xcode template headers.
```suggestion
import Logging
```